### ISSUE TITLE
Batch import

### ISSUE THREAD

**Issue by sheepyhollow** [PhysikOnline-FFM/gu-urlshorter#27]
Am Beispiel des SSC habe ich gesehen, dass die riesige Listen von Verlinkungen haben, die sie demnächst anlegen wollen. Also: Alle derzeit aktiven Studienordnungen mit goethe.links verlinken etc.

Batch import muss kein Feature sein - man könnte Powerusern sowas aber als Service anbieten. Eigentlich sollte es doch nicht unmöglich sein, eine csv direkt in die Datenbank zu schubsen?

**Comment by svenk** [comment]
Ja, eine CSV im richtigen Format sollte sich einfach verarbeiten lassen (sobald man das implementiert hat!). Das ist aber manchmal dem Benutzer schwierig zu verklickern, dass das wirklich eine ganz einfach Excel-Tabelle sein muss in der in der ersten Reihe die Spaltennamen stehen und dann untendrunter die Einträge.

**Comment by sheepyhollow** [comment]
> ist aber manchmal schwierig zu verklickern

sehe ich, wie gesagt, auch eher als ein betreutes Premium-Feature ("Wenn Sie eine große Zahl von Links anlegen wollen, melden Sie sich bitte bei ..."). Kann man sich ja auch bezahlen lassen :grin:
